### PR TITLE
Add websites to crawling.

### PR DESCRIPTION
**Pull request by liuche**
This adds websites into providers under:
venues/details/<place-id>/providers/google/website/<website>

### CODE REVIEW

File: scripts/prox_crosswalk.py [hunk 1]
```diff
@@ -34,7 +34,7 @@
 _YELP_ID_TO_PLACE_CACHE = {}
 
 def getAndCacheProviderIDs(keyID, providersList, identifiers):
-    providerIDs = _getCachedIDsForPlace(keyID, providersList)
+    providerIDs = {k: v for k,v in _getCachedIDsForPlace(keyID, providersList).items() if k in providersList}
```
Comment (mcomella) [inline scripts/prox_crosswalk.py]: Is the motivation behind this addition to:

1. Ensure `k in providersList`?
1. Create a copy?

I believe 1 is taken care of inside `_getCachedIDsForPlace` and I don't think 2 is necessary.

File: app/representation.py [hunk 2]
```diff
@@ -84,6 +84,14 @@ def updateRecord(yelpID, **details):
         "images": _imageRecords(info["images"], info["url"])
       }
 
+    if "google" in details:
+      info = details["google"]
+      # No error, just no relevant website info
+      val = None
+      if "website" in info:
+          val = { "website": info["website"] }
```
Comment (mcomella) [inline app/representation.py]: nit: Could simplify with `info.get('website', None)`

File: scripts/crawl_expand.py [hunk 3]
```diff
@@ -84,6 +83,7 @@ def makeNewStatusTable(config, updatedProviders, placeProviderIDs, newProviders)
 
 if __name__ == '__main__':
     TEST_CONFIG = { "yelp3": 3,
-                    "tripadvisor": 3 }
+                    "tripadvisor": 3,
+                    "google": 3 }
 
-    expandPlaces(TEST_CONFIG, GPS_LOCATIONS["CHICAGO_CENTER"], 3)
+    expandPlaces(TEST_CONFIG, GPS_LOCATIONS["CHICAGO_CENTER"], 30)
```
Comment (mcomella) [inline scripts/crawl_expand.py:89]: Is committing 3 -> 30 intentional? (I'm not sure over how large an area we should be searching)

**Review by mcomella — APPROVED** [review]

File: scripts/prox_crosswalk.py (hunk 1 above)
Comment (liuche) [reply scripts/prox_crosswalk.py]: Oh good call, I think I added this while debugging the mysterious for loop bug, and forgot to take it out. Will undo.

File: scripts/crawl_expand.py (hunk 3 above)
Comment (liuche) [reply scripts/crawl_expand.py:89]: Yep, I don't remember why I switched it to 3, bc the crawl script should by default crawl the 30 km circle of chicago - probably a mis-commit from before, so I'm undoing it now.

**Pull request merged**